README.md

```diff
@@ -253,6 +246,7 @@ cmake -G Ninja .. \
     -DLLVM_PARALLEL_LINK_JOBS=4 \
     -DLLVM_PARALLEL_COMPILE_JOBS=4
 ninja
+sudo cp <workdir>/circt/build/bin/circt-opt /usr/bin/circt-opt
```

Comment:
What if user already has specific version of `circt-opt` in `/usr/bin`? It is a bad idea to overwrite it in such a way.
Again, AFAIK, it is possible here to use this binary from it's `CIRCT_DIR` location.